local/nginx/include/metrics.conf

```diff
@@ -45,3 +45,5 @@ location /postgres_metrics {
     proxy_pass http://postgres_metrics/metrics;
     proxy_pass_request_headers  on;
 }
+
+#  TODO: Add metrics for pow-auth-server
```

Comment:
`/metrics` endpoint is already being exposed by the pow-auth-server if we wanted to expose it here

Reply:
Okay, will do that. Anyway, in the server, we are not creating any custom metric.